Article/Article.less

```diff
@@ -39,6 +39,9 @@
     transform: translate(-50%);
     background-color: white;
   }
+  .hidden {
+    display: none !important;
```

Comment:
I'm curious as to why you had to use the important on this one. I'm assuming while you were toggling you weren't removing the other class that displayed the content. In that case you might want to think about removing that class from the classList.